src/main/java/com/openlattice/chronicle/services/ChronicleServiceImpl.java

```diff
@@ -1136,7 +1151,7 @@ public void refreshStudyInformation() {
         SearchApi searchApi;
         PrincipalApi principalApi;
         try {
-            ApiClient apiClient = apiClientCache.get( ApiClient.class );
+            ApiClient apiClient = intApiClientCache.get( ApiClient.class );
```

Comment:
This is a background job, so it doesn't have to be `prodApiClientCache`, but it also doesn't write any data, so maybe it should be `prodApiClientCache`... what do you think?

Reply:
yeah, I agree, it's not a lot of data.